api/src/controllers/website.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * This module contains handlers for the "clientWebsite" route.
+ * @module website
+ */
+const axios = require('axios');
+
+const Website = require('../models/website.model');
+
+const { fillErrorObject } = require('../middleware/error');
+
+var ObjectId = require('mongoose').Types.ObjectId;
```

Comment:
use `const` instead of `var`: https://ui.dev/var-let-const/

Reply:
Also, this variable may not need to be global. Prefer reducing the scope.

Reply:
Turns I don't need ObjectId all together, so removing it... 😂